### PR TITLE
v: add page

### PR DESCRIPTION
**Pull request by dy-tea**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
V 0.4.8 9acf058

### CODE REVIEW

File: pages/common/v.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# v
+
+> A tool for managing V source code.
+> More information: <https://docs.vlang.io/getting-started.html>.
+
+- Compile a file and output its executable:
+
+`v {{/path/to/file.v}}`
```
Comment (gutjuri) [inline pages/common/v.md:8]:
```suggestion
`v {{path/to/file.v}}`
```

This can be a relative path, right?

File: pages/common/v.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Compile a file and [run] it, output its executable:
+
+`v crun {{path/to/file.v}}`
+
+- Re-compile on every modification to a file:
+
+`v watch {{path/to/file.v}}`
+
+- Re-[run] on every modification to a file:
```
Comment (gutjuri) [inline pages/common/v.md:26]:
```suggestion
- Compile a file and run it:

`v run {{path/to/file.v}}`

- Compile a file and run it, output its executable:

`v crun {{path/to/file.v}}`

- Re-compile on every modification to a file:

`v watch {{path/to/file.v}}`

- Re-run on every modification to a file:
```

Usually, we only use mnemonics when the short form of an argument appears in the example.

**Review by gutjuri — COMMENTED** [review]
Thanks! I have two suggestions.

**Review by gutjuri — APPROVED** [review]
LGTM, thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**